lib/fs.js

```diff
@@ -1213,7 +1213,9 @@ function FSWatcher() {
   this._handle.onchange = function(status, event, filename) {
     if (status < 0) {
       self._handle.close();
-      self.emit('error', errnoException(status, 'watch'));
+      var error = errnoException(status, 'watch ' + filename);
```

Comment:
Can you use `let` instead of `var`, and a template string instead of concatenation.